connector/connect/common/src/main/buf.gen.yaml

```diff
@@ -16,21 +16,21 @@
 #
 version: v1
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/cpp:v3.20.0-1
+  - plugin: buf.build/protocolbuffers/cpp:v22.3
     out: gen/proto/cpp
-  - remote: buf.build/protocolbuffers/plugins/csharp:v3.20.0-1
+  - plugin: buf.build/protocolbuffers/csharp:v22.3
     out: gen/proto/csharp
-  - remote: buf.build/protocolbuffers/plugins/java:v3.20.0-1
+  - plugin: buf.build/protocolbuffers/java:v22.3
     out: gen/proto/java
   - remote: buf.build/grpc/plugins/ruby:v1.47.0-1
     out: gen/proto/ruby
-  - remote: buf.build/protocolbuffers/plugins/ruby:v21.2.0-1
+  - plugin: buf.build/protocolbuffers/ruby:v22.3
     out: gen/proto/ruby
    # Building the Python build and building the mypy interfaces.
-  - remote: buf.build/protocolbuffers/plugins/python:v3.19.3-1
+  - plugin: buf.build/protocolbuffers/python:v22.3
     out: gen/proto/python
-  - remote: buf.build/grpc/plugins/python:v1.47.0-1
+  - plugin: buf.build/grpc/python:v1.54.1
     out: gen/proto/python
-  - name: mypy
+  - plugin: buf.build/community/nipunn1313-mypy:v3.4.0
```

Comment:
I presume the mypy plugin is in referenced to the one maintained here: 

https://github.com/nipunn1313/mypy-protobuf

If so, this is now also a Remote Plugin available at:

https://buf.build/community/nipunn1313-mypy